go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/gboddin/go-www-authenticate-parser v0.0.0-20230926203616-ec0b649bb077
 	github.com/google/go-containerregistry v0.19.2
 	github.com/olekukonko/tablewriter v0.0.5
+	github.com/rogpeppe/go-internal v1.10.0
```

Comment:
do we want to use this package and not golang.org/x/mod/semver?

Reply:
changed